OrangeHRMTests/Data/Constants/DashboardPageConstants.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace OrangeHRMTests.Data.Constants
+{
+    public class DashboardPageConstants
+    {
+        public const string DashboardPageUrl = "https://opensource-demo.orangehrmlive.com/web/index.php/dashboard/index";
+    }
```

Comment:
it's better to store all Urls in testsettings

Reply:
Fixed